.github/workflows/scanner.yml

```diff
@@ -37,3 +39,48 @@ jobs:
           severity: CRITICAL,HIGH
           exit-code: 1
           ignore-unfixed: true
+
+  license:
+    runs-on: ubuntu-24.04-ppc64le
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v5
+      
+      - name: Build image
+        working-directory: ${{ inputs.path }}
+        run: |
+          make build REGISTRY=${{ env.ICR_REGISTRY }}/${{ env.ICR_NAMESPACE }}
+      
+      - name: Save image in file
+        run: |
+          IMAGE_NAME=$(podman images --format "{{.Repository}}:{{.Tag}}" | grep "${{ env.ICR_REGISTRY }}/${{ env.ICR_NAMESPACE }}" | head -n 1)
+          podman save > ${{ env.IMAGE_FILE_NAME }} $IMAGE_NAME
+
+      - name: Generate trivy sbom scan report
+        uses: aquasecurity/trivy-action@0.33.1
+        with:
+          input: ${{ env.IMAGE_FILE_NAME }}
+          format: cyclonedx
+          output: ${{ env.TRIVY_SBOM_FILE }}
+      
+      - name: Enrich trivy SBOM with parlay
+        run: |
+          mkdir -p $HOME/bin
+          wget https://github.com/snyk/parlay/releases/download/v0.6.4/parlay_Linux_ppc64le.tar.gz
```

Comment:
Why are we not using the latest version?
Wondering how we can monitor version upgrades.